source/frontend/src/features/mapSideBar/acquisition/tabs/agreement/common/UpdateAcquisitionAgreementView.tsx

```diff
@@ -0,0 +1,118 @@
+import axios, { AxiosError } from 'axios';
+import { Formik } from 'formik';
+import styled from 'styled-components';
+
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import SidebarFooter from '@/features/mapSideBar/shared/SidebarFooter';
+import { getCancelModalProps, useModalContext } from '@/hooks/useModalContext';
+import { IApiError } from '@/interfaces/IApiError';
+import { ApiGen_Concepts_Agreement } from '@/models/api/generated/ApiGen_Concepts_Agreement';
+
+import AcquisitionAgreementForm from '../form/AcquisitionAgreementForm';
+import { AcquisitionAgreementFormYupSchema } from '../form/AcquisitionAgreementFormYupSchema';
+import { AcquisitionAgreementFormModel } from '../models/AcquisitionAgreementFormModel';
+
+export interface IUpdateAcquisitionAgreementViewProps {
```

Comment:
On the name of the component, usually views only display data while forms contain editable sections.
In this case it would be UpdateAcquisitionAgreementForm

Reply:
Updated